tsconfig.json

```
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "baseUrl": "./",
     "outDir": "./dist",
-    "sourceMap": true,
+    "sourceMap": false,
```

```suggestion
    "sourceMap": true,
```

Find a different way to solve this